src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -100,6 +100,10 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                         "ThreadCount must be positive.");
                 }
 
+                if (!Constants.ECONNRESET.HasValue)
+                {
+                    _logger.LogWarning("Unable to determine ECONNRESET value on this platform.");
+                }
```

Comment:
I copied the formatting directly from 1.0.0. You wrote that, right? 😝 I think having them together is fine since they're so closely related.